tests/monster_test_generated.rs

```diff
@@ -93,6 +95,19 @@ impl std::fmt::Debug for InParentNamespace<'_> {
       ds.finish()
   }
 }
+#[non_exhaustive]
```

Comment:
non_exhaustive on struct?

Reply:
That annotation should not be on Flatbuffers structs. InParentNameSpace is a table.
If you mean on Rust structs, it disables exhaustive pattern matching so you have to add the `..` pattern.